grafana/common/PG_Overview.json

```diff
@@ -441,5 +439,5 @@
   "timezone": "browser",
   "title": "PostgreSQL Overview",
   "uid": "pxinDnNik",
-  "version": 2
+  "version": 1
```

Comment:
Is this version something that should be only increasing or is it more-or-less the version *for* the specific `pluginVersion` (say), so not significant?

Reply:
It's basically the version that's kept track of within Grafana so it can keep track of history. It's incremented up every time you save a dashboard within Grafana. But if different people are importing/exporting or even just doing it at different times during development, it can easily get reset. Not really a big deal when using provisioning.